org.eclipse.mylyn-site/pom.xml

```diff
@@ -93,7 +93,7 @@
                     -root ${project.build.directory}/mylyn-sync
                     -relative ${org.eclipse.justj.p2.manager.relative}
                     -version-iu org.eclipse.mylyn.sdk_feature.
-                    -iu-filter-pattern org.eclipse.mylyn.*
+                    -iu-filter-pattern org.eclipse.mylyn.*|org.eclipse.egit.mylyn.*|org.eclipse.egit.github.*|org.eclipse.cdt.mylyn|org.eclipse.cdt.mylyn.*
```

Comment:
Great! This was my comment [for the very first version of the manifest](https://github.com/eclipse-mylyn/org.eclipse.mylyn/pull/123/files#r1186682613)